pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -231,16 +232,17 @@ let
       "--cxxopt=-x" "--cxxopt=c++" "--host_cxxopt=-x" "--host_cxxopt=c++"
     ];
 
+    # We intentionally overfetch so we can share the fetch derivation across all the different configurations.
     fetchAttrs = {
-      sha256 =
-        if cudaSupport then
-          "sha256-n8wo+hD9ZYO1SsJKgyJzUmjRlsz45WT6tt5ZLleGvGY="
-        else {
-          x86_64-linux = "sha256-A0A18kxgGNGHNQ67ZPUzh3Yq2LEcRV7CqR9EfP80NQk=";
-          aarch64-linux = "sha256-mU2jzuDu89jVmaG/M5bA3jSd7n7lDi+h8sdhs1z8p1A=";
-          x86_64-darwin = "sha256-9nNTpetvjyipD/l8vKlregl1j/OnZKAcOCoZQeRBvts=";
-          aarch64-darwin = "sha256-dOGUsdFImeOLcZ3VtgrNnd8A/HgIs/LYuH9GQV7A+78=";
-        }.${stdenv.system} or (throw "unsupported system ${stdenv.system}");
```

Comment:
Oh God, are we really rid of this? 😍

Reply:
yessir!! s/o to @uri-canva !

Reply:
Ouch. I guess we'll need to a nixpkgs-review then

Reply:
On x86_64-linux the `jaxlib`, `jaxlibWithCuda`, and `jax` derivations all built fine for me. (I did have to fiddle with the hash)

Since aarch64-darwin has never been supported by the source build, and x86_64-darwin support is waning, presumed to be broken now, I think x86-64_linux is the only platform that we're really supporting the source build on atm. Do we really need a nixpkgs-review then?

Reply:
...well, it offloads the mental load to the machine

Reply:
Ah @uri-canva you are correct: ofborg is failing https://gist.github.com/GrahamcOfBorg/819dce99401b0659eb259dd2e6d6e752

Any objections to marking aarch64-linux as broken? Or perhaps removing it from `meta.platforms`?

Reply:
I'm a bit lost now, why is "attribute 'aarch64-linux' missing", and was it missing before?

Reply:
On a second look, we are already specifying `meta.platforms` correctly

https://github.com/NixOS/nixpkgs/blob/37e8b04cc76a2af79994b2b4439bf67e00559082/pkgs/development/python-modules/jaxlib/bin.nix#L128

But for some reason ofborg is still trying to evaluate the derivation. I opened [a discussion on Discourse](https://discourse.nixos.org/t/ofborg-does-not-respect-meta-platforms/27019) to track.